src/capture-screenshot.ts

```diff
@@ -59,7 +58,7 @@ export async function captureScreenshot(options: CaptureScreenShotOptions) {
   const page = await browser.newPage();
 
   page.on("error", (error) => {
-    console.log(`🚨 ${error}`);
+    console.log(`🚨  Page Error: ${error}`);
```

Comment:
I noticed we had inconsistent white spacing in these outputs so I made it consistent. Now it's emoji then two spaces.